src/olympia/devhub/templates/devhub/addons/submit/base.html

```diff
@@ -13,6 +13,12 @@
 {% endblock %}
 
 {% block content %}
+{% if submit_notification_warning %}
+    <div class="notification-box warning">
+        {# The submit notification warning is controlled by admins, HTML is allowed #}
+        {{ submit_notification_warning|safe }}
+    </div>
+{% endif %}
```

Comment:
Yeah, I wasn't sure it really deserved a test, but I can add a simple one testing that HTML works.